moler/io/raw/terminal.py

```diff
@@ -36,6 +38,8 @@ def __init__(self, moler_connection, cmd="/bin/bash", select_timeout=0.002,
         :param dimensions: dimensions of the psuedoterminal
         """
         super(ThreadedTerminal, self).__init__(moler_connection=moler_connection)
+        self.debug_hex_on_non_printable_chars = False  # Set True to log incoming non printable chars as hex.
+        self.debug_hex_on_all_chars = False  # Set False to log incoming data as hex.
```

Comment:
Should be 'Set **True** to log incoming data as hex.'

Reply:
Done